### DISCUSSION TITLE
How to handle zod parse error from api/endpoint declaration

### DISCUSSION THREAD

**Discussion by fmartins-andre** [ecyrbe/zodios#498] [Q&A]
The schema validation errors are shown in the console when in development mode, but in production, no error is emitted.
I wonder if there's a way for handle these kind of error from a central location, link in the api ou endpoint file, allowing me to toast some message or handle the error in some way.

**Comment by ecyrbe** [comment]
Hello, 
Don't mind me converting this to discussion, also if you need help you can seek help in our discord Channel.
